### PR TITLE
options for computation

### PR DESCRIPTION
**Pull request by yichen88**

### CODE REVIEW

File: computation/src/main/java/com/powsybl/computation/ComputationConfigBuilder.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+public class ComputationConfigBuilder {
+
+    private final Map<String, Integer> timeoutMap = new HashMap<>();
```
Comment (geofjamg) [inline computation/src/main/java/com/powsybl/computation/ComputationConfigBuilder.java]: Timeout is redondant with one defined in the command. I think you should remove it.

Comment (yichen88) [reply computation/src/main/java/com/powsybl/computation/ComputationConfigBuilder.java]: Maybe we should deprecated the timeout of the command. The estimated time of same computation could be different based on the platform.

Comment (yichen88) [reply computation/src/main/java/com/powsybl/computation/ComputationConfigBuilder.java]: And more, the timeout could be configued with qos in a file and loaded into ComputationOptions.

**Comment by yichen88** [comment]
I think there are two use cases about the time limit.
1. to avoid an infinite loop, getTimeout() @geofjamg 
2. to avoid waiting too long. getDeadline() @sylvlecl

**Comment by yichen88** [comment]
#736

**Pull request closed without merging**